lib/ansible/parsing/yaml/objects.py

```diff
@@ -53,7 +54,17 @@ def _set_ansible_position(self, obj):
     ansible_pos = property(_get_ansible_position, _set_ansible_position)
 
 
-class AnsibleMapping(AnsibleBaseYAMLObject, dict):
+# try to always use orderddict with yaml, after py3.6 the dict type already does this
+odict = dict
+if PY2 or sys.version_info[:2] < (3, 7):
```

Comment:
Why not simplify this?

```suggestion
if sys.version_info[:2] < (3, 7):
```

Reply:
cause when i wrote it i thought only >=3.7 always defaulted to ordered dicts